index.js

```diff
@@ -17,4 +17,7 @@ module.exports = fp((fastify, opts, next) => {
   fastify.addHook('onClose', (fastify, done) => fastify.ws.close(done))
 
   next()
+}, {
+  fastify: '>=1.x',
```

Comment:
Does `>=1.x` mean something different than `1.x`?

Reply:
yep. [>=1.x](https://github.com/npm/node-semver#ranges) will match anything greater or equal than 1.0.0 while [1.x](https://github.com/npm/node-semver#x-ranges-12x-1x-12-) will match the `>=1.0.0 <2.0.0` range. May I change it to `1.x` ?

Reply:
Ah yeah sorry, wasn't at all clear with the question — meant to ask whether you thought it was worthwhile version-locking to the `1.x` branch of fastify and then verifying compatibility with 2.x before relaxing the version lock. I'm leaning towards yes, just to be safe. Want to make that change and then I'll merge this in and cut a new release?

I think it'll have to be a 1.0.0 release of `fastify-ws` b/c I'd imagine that new version lock (to `fastify '>=1.x'`) would prevent compatibility with pre-1.x versions of fastify. Does that sound reasonable?

Reply:
Also sorry for taking so long to initially look at this — I think I have notifications working on this repo now, so I should be more responsive going forward 😄

Reply:
Tomorrow's going to be a busy day and I have some time now, so I'm gonna merge your changes in as they currently stand and then restrict the version to `1.x` in a separate commit.

Reply:
Sorry, i went to sleep yesterday. I rather not encourage to support the 0.x relases of fastify, because it introudeced breakng changes frequently, and I have no idea how this plugin plays together with those breaking changes. But as temporary solution setting it to `<2.x` would make the trick. Thanks for merging it. :+1:

Reply:
Totally agree. Also I took your commit info to add you to the contributors list, but if you'd like to adjust it to read differently go ahead and submit a PR 🙂 

Thanks again!